NGS2_WITNESS_Cycle1_MASTER.R

```diff
@@ -0,0 +1,59 @@
+#Created by Pablo Diego Rosell, PhD, for Gallup inc. on August 2017

```

Comment:
This file (`NGS2_WITNESS_Cycle1_MASTER.R`) needs to be backed out of this PR. If this gets merged, then it'll overwrite necessary changes we have in `master` to begin with.